tidb-cloud/connected-care-detail.md

```diff
@@ -0,0 +1,157 @@
+    <td>-</td>
+    <td>S2: 8 business hours</td>
+    <td>S2: 2 hours</td>
+    <td>S2: 1 hour</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>S3: 12 business hours</td>
+    <td>S3: 8 business hours</td>
+    <td>S3: 4 business hours</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>S4: 24 business hours</td>
+    <td>S4: 24 business hours</td>
+    <td>S4: 24 business hours</td>
+  </tr>
+  <tr>
+    <td rowspan="10">Services</td>
+    <td>Billing and account support</td>
+    <td>Billing and account support</td>
+    <td>Billing and account support</td>
+    <td>Billing and account support</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>Ticket-based technical support</td>
+    <td>Ticket-based technical support</td>
+    <td>Ticket-based technical support</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>End-to-end database guidance</td>
+    <td>End-to-end database guidance</td>
+    <td>End-to-end database guidance</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: AI chat in IM</td>
+    <td>Connected: AI chat in IM</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: IM subscription for TiDB Cloud alerts</td>
+    <td>Connected: IM subscription for TiDB Cloud alerts</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: IM subscription for support ticket updates</td>
+    <td>Connected: IM subscription for support ticket updates</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: IM ticket creation</td>
+    <td>Connected: IM ticket creation</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: Clinic service for advanced monitoring</td>
+    <td>Connected: Clinic service for advanced monitoring</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>-</td>
+    <td>Connected: IM interaction for support tickets</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>-</td>
+    <td>-</td>
+    <td>Technical Account Management</td>
+  </tr>
+  <tr>
+    <td rowspan="5">Customer Resources</td>
+    <td>Documentations</td>
+    <td>Documentations</td>
+    <td>Documentations</td>
+    <td>Documentations</td>
+  </tr>
+  <tr>
+    <td>FAQ</td>
+    <td>FAQ</td>
+    <td>FAQ</td>
+    <td>FAQ</td>
+  </tr>
+  <tr>
+    <td>Community Workspace</td>
+    <td>Community Workspace</td>
+    <td>Community Workspace</td>
+    <td>Community Workspace</td>
+  </tr>
+  <tr>
+    <td>TiDB.AI</td>
+    <td>TiDB.AI</td>
+    <td>TiDB.AI</td>
+    <td>TiDB.AI</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>Knowledge Base</td>
+    <td>Knowledge Base</td>
+    <td>Knowledge Base</td>
+  </tr>
+</tbody>
+</table>
+
+- Business Hours: 9 AM - 6 PM local time, Monday to Friday
+- IM: Instant Message. Currently, [Slack](https://slack.com/) and [Lark](https://www.larksuite.com/) are supported. In the future, Microsoft Teams is planned.
+- Clinic: a modern monitoring service that provides advanced monitoring and diagnostic capabilities for your TiDB Cloud clusters.
```

Comment:
Do we need to provide the Clinic link here https://docs.pingcap.com/tidb/stable/clinic-introduction#pingcap-clinic-overview?

Reply:
It refers to Clinic on TiDB Cloud (https://github.com/pingcap/docs/pull/19895/files), which will be released along with the Connected Care service.